migration/i7.1z/postgresql/202402020915_IDEMPIERE-6028.sql

```diff
@@ -0,0 +1,39 @@
+-- Adding Override User field to Workflow Responsible User from Client Side
+-- 02-Feb-2024, 8:59:49 PM IST
+INSERT INTO AD_Element (AD_Element_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,ColumnName,Name,PrintName,EntityType,AD_Element_UU) VALUES (203907,0,0,'Y',TO_TIMESTAMP('2024-02-02 20:59:48','YYYY-MM-DD HH24:MI:SS'),0,TO_TIMESTAMP('2024-02-02 20:59:48','YYYY-MM-DD HH24:MI:SS'),0,'Override_ID','Overridden WF Responsible','Overridden WF Responsible','D','6c2430de-745a-4894-80da-42118699f186')
+;
+
+-- 02-Feb-2024, 9:00:43 PM IST
+INSERT INTO AD_Reference (AD_Reference_ID,Name,ValidationType,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,EntityType,IsOrderByValue,AD_Reference_UU) VALUES (200265,'AD_WF_Responsible','T',0,0,'Y',TO_TIMESTAMP('2024-02-02 21:00:42','YYYY-MM-DD HH24:MI:SS'),0,TO_TIMESTAMP('2024-02-02 21:00:42','YYYY-MM-DD HH24:MI:SS'),0,'D','N','87e21ecf-cbf6-4034-b06d-2876f1c0c080')
+;
+
+-- 02-Feb-2024, 9:01:14 PM IST
+INSERT INTO AD_Ref_Table (AD_Reference_ID,AD_Table_ID,AD_Key,AD_Display,WhereClause,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,IsValueDisplayed,EntityType,AD_Ref_Table_UU) VALUES (200265,646,10483,10475,'AD_WF_Responsible.AD_Client_ID=0',0,0,'Y',TO_TIMESTAMP('2024-02-02 21:01:14','YYYY-MM-DD HH24:MI:SS'),0,TO_TIMESTAMP('2024-02-02 21:01:14','YYYY-MM-DD HH24:MI:SS'),0,'N','D','b4f0db4c-438d-43d9-bafa-316306726ded')
```

Comment:
@CarlosRuiz-globalqss  I think those can be overwritten as ultimately a tenant admin decide who will be a responsible in his company.

Reply:
> @CarlosRuiz-globalqss I think those can be overwritten as ultimately a tenant admin decide who will be a responsible in his company.

Then you need to define how every overwrite would work.

For example the error I reported here below: https://github.com/idempiere/idempiere/pull/2469#discussion_r1798472724 is overriding a Role(System) by Human(Tenant) - that created a bug.

As I see there would be 20 possible combinations, and you need to test and check implications of every one:

| System  | Tenant  | Comment                               |
|---------|---------|---------------------------------------|
| Human   | Invoker | this doesn’t sound possible to define |
| Human   | Manual  |                                       |
| Human   | Org     |                                       |
| Human   | Role    |                                       |
| Invoker | Human   |                                       |
| Invoker | Manual  |                                       |
| Invoker | Org     |                                       |
| Invoker | Role    |                                       |
| Manual  | Human   |                                       |
| Manual  | Invoker |                                       |
| Manual  | Org     |                                       |
| Manual  | Role    |                                       |
| Org     | Human   |                                       |
| Org     | Invoker |                                       |
| Org     | Manual  |                                       |
| Org     | Role    |                                       |
| Role    | Human   |                                       |
| Role    | Invoker | bug here with actual code             |
| Role    | Manual  |                                       |
| Role    | Org     |                                       |

Reply:
@CarlosRuiz-globalqss Let me test this all scenario. But I think this all override should allow so that tenant can define who are approver and how approver works. 
If there is bug then we can restrict or fix bug.